db/models/cases.js

```diff
@@ -37,31 +37,45 @@ class Service extends BaseService {
     throw new Error('Could not publish the case. Make sure all are moved into staging state.')
   }
 
+  /**
+   * Consent To Publish
+   *
+   * @method consentToPublishing
+   * @param {Number} case_id
+   * @return {Object}
+   */
+  async consentToPublishing(case_id) {
+    const result = await this.updateOne(case_id, { consented_to_publishing_at: new Date() });
+    if (result){
+      return this._mapCase(result);
+    }
+  }
+
   /**
    * Mark Case Staging
    *
-   * @method stage
+   * @method moveToStaging
    * @param {Number} case_id
-   * @return {Array}
+   * @return {Object}
    */
   async moveToStaging(case_id) {
-    const results = await this.updateOne(case_id, { state: 'staging' });
-    if (results) {
-      return this._mapCase(results);
+    const result = await this.updateOne(case_id, { state: 'staging' });
```

Comment:
This is currently in conflict with dev, don't forget to keep the `staged_at` change from dev as you resolve the conflict.